Cargo.toml

```diff
@@ -1,52 +1,37 @@
 [package]
-name = "{{project-name}}"
-version = "0.1.0"
+name = "dfns-cggmp21-blueprint"
+version = "0.1.1"
+description = "A DFNS CGGMP21 Blueprint that can run keygen and signing jobs on demand from the Tangle network"
 edition = "2021"
-description = "{{project-description}}"
-authors = ["{{authors}}"]
-license = "MIT OR Apache-2.0"
-homepage = "{{project-homepage}}"
-repository = "https://github.com/{{gh-username}}/{{project-name}}"
-readme = "README.md"
-categories = ["cryptography", "cryptography::cryptocurrencies"]
-keywords = ["tangle", "blueprint", "avs"]
-rust-version = "1.81"
 
-[dependencies]
-tracing = "0.1"
-async-trait = "0.1"
-color-eyre = "0.6"
-structopt = "0.3.26"
-tokio = { version = "^1", default-features = false, features = ["full"] }
-tracing-subscriber = { version = "0.3", features = ["parking_lot", "env-filter"] }
+[package.metadata.blueprint]
+manager = { Evm = "DfnsBlueprint" }
 
-[dependencies.gadget-sdk]
-version = "0.4.0"
-default-features = false
-features = ["getrandom"]
+[dependencies]
+#gadget-sdk = { git = "https://github.com/tangle-network/gadget/", features = ["std"] }
+gadget-sdk = { path = "../gadget/sdk", features = ["std"] }
+color-eyre = { version = "0.6", features = ["tracing-error", "color-spantrace"] }
+tokio = { version = "1.40", default-features = false, features = ["full"] }
+hex = { version = "0.4.3", default-features = false }
+k256 = { version = "0.13.3", default-features = false }
+bincode = "1.3.3"
+sp-core = { version = "31.0.0", default-features = false }
+cggmp21 = { git = "https://github.com/LFDT-Lockness/cggmp21", features = ["curve-secp256k1"] }
```

Comment:
Why do you use cggmp21 as git dependency? It is released on crates.io: https://crates.io/crates/cggmp21